stix_shifter_modules/aws_athena/stix_translation/json/ocsf_from_stix_map.json

```diff
@@ -77,6 +162,58 @@
       ]
     }
   },
+  "process": {
+    "fields": {
+      "binary_ref.name": [
+        "file.name"
+      ],
+      "command_line": [
+        "process.cmd_line"
+      ],
+      "created": [
+        "process.created_time"
+      ],
+      "extensions.'x-ocsf-process-ext'.integrity": [
+        "process.integrity"
+      ],
+      "extensions.'x-ocsf-process-ext'.integrity_id": [
+        "process.integrity_id"
+      ],
+      "extensions.'x-ocsf-process-ext'.is_system": [
+        "process.is_system"
+      ],
+      "extensions.'x-ocsf-process-ext'.lineage": [
+        "process.lineage"
+      ],
+      "extensions.'x-ocsf-process-ext'.loaded_modules": [
+        "process.sandbox"
+      ],
+      "extensions.'x-ocsf-process-ext'.terminated_time": [
+        "terminated_time"
+      ],
+      "extensions.'x-ocsf-process-ext'.tid": [
+        "process.tid"
+      ],
+      "extensions.'x-ocsf-process-ext'.uid": [
+        "process.uid"
```

Comment:
This should be `process.x_unique_id` like we have for some of the other connectors https://github.com/opencybersecurityalliance/stix-shifter/blob/7b015f15d4f14eac6aee58ecb964927fa8fbc20b/stix_shifter_modules/reaqta/stix_translation/json/to_stix_map.json#L53